Reply/app/src/main/java/com/materialstudies/reply/ui/home/HomeScreen.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.materialstudies.reply.ui.home
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import com.google.accompanist.themeadapter.material.MdcTheme
+import com.materialstudies.reply.data.Email
+import com.materialstudies.reply.data.EmailStore
+import com.materialstudies.reply.ui.common.EmailList
+import com.materialstudies.reply.util.DevicePreviews
+import com.materialstudies.reply.util.ThemePreviews
+
+@Composable
+fun HomeScreen(
+    emails: List<Email>,
+    modifier: Modifier = Modifier,
+    onNavigateToEmail: (Long) -> Unit = {},
+    onEmailLongClick: () -> Unit = {},
```

Comment:
Should this event handler also pass in an email ID so you know which email was long clicked?

Reply:
The original implementation is the same for any email - doesn't need an ID for anything, it just shows a generic bottom sheet